demos/render/assets/img_render_lvgl_logo_l8.c

```diff
@@ -0,0 +1,66 @@
+
+#if defined(LV_LVGL_H_INCLUDE_SIMPLE)
+#include "lvgl.h"
+#elif defined(LV_BUILD_TEST)
+#include "../lvgl.h"
```

Comment:
Use an include similar to the other images.

Reply:
Fixed.